src/app/events/[slug]/page.tsx

```diff
@@ -59,3 +65,20 @@ export default function EventEntry({ params }: EventProps) {
     </>
   )
 }
+
+function buildCtaArray(
```

Comment:
Should we move the function in a util file? In case we want to reuse it?

Reply:
In a local `utils` folder?

Reply:
Yes! Good idea. Thank you.